code/controllers/subsystem/religion.dm

```diff
@@ -3,10 +3,8 @@ SUBSYSTEM_DEF(religion)
 	init_order = 19
 	flags = SS_NO_FIRE|SS_NO_INIT
 
-	var/bible_deity_name
-	var/Bible_icon_state
-	var/Bible_item_state
-	var/Bible_name
-	var/Bible_deity_name
-
-	var/holy_weapon
+	var/religion
+	var/deity
+	var/bible_name
+	var/list/bible_icons = list("icon_state", "item_state")
```

Comment:
there's no advantage to this.
you're:
1. adding a slowdown by defining a list in the definition
2. adding some memory costs for maintaining the list, meaning the cost of the strings is 2 strings + a list instead of just 2 strings, the advantages of lists is looping and random access, neither are needed for bibles.